### ISSUE TITLE
No echo in console.bat

### ISSUE THREAD

**Issue by nin-jin** [orientechnologies/orientdb#2446]
When i types anything console prints nothing.
Then i press enter and typed command are executed.

Command history not works (by up an down arrows).
Context menu not opens.

Windows 8.1

**Comment by yammelvin** [comment]
I am experiencing exactly the same thing.
I am using Windows 7 and Java 8
The console does not echo anything I type before pressing "Enter", therefore I cannot see anything I type.

Please fix the bug.

**Comment by yammelvin** [comment]
I think I found the problem.

I had sh.exe (part of MSYS) in my path. In other words, I can run "sh" in Windows Command Prompt.
The OGremlinConsole.java or OConsoleDatabaseApp.java calls stty() which in turns call "sh" command, assuming Windows does not have the "sh" command, then it disable character encoding.

The moment I removed MSYS from my path, ie. no more "sh" command in Windows Command Prompt, the console enables character echoing.
Now I can see what I type.
